Week1/homework/js-exercises/animals.js

```diff
@@ -0,0 +1,7 @@
+let items = [];
+console.log("This is an empty array");
+console.log(items)
+let animals = ["Lion","Tiger","Monkey"];
```

Comment:
Use proper spacing. One space after each comma. I believe VSCode should have autoformatting tools